### PR TITLE
Socialisation

### PR DESCRIPTION
**Pull request by Charlotte-Morgan**
Update files, add landcover icon and definition
objective.rst
introduction.rst
inasafe_concepts.rst
introduction to QGIS.rst
run basic inasafe.rst

Other edits required - update screenies and fix links
Update datasets.rst with version 3.4 keywords

@adissadis @easmetz please check

### CODE REVIEW

File: docs/source/training/socialisation/inasafe_concepts.rst [hunk 1]
```diff
@@ -329,7 +331,29 @@ in the globe produced in a standardised and rigorous way.
 +--------------+---------------------------------------------------------------+
 |Notes         | Make sure you know if your data represent density or count    |
 +--------------+---------------------------------------------------------------+
-|Source        | WorldPop or from a  national census agency                    |
+|Source        |Can be obtained from a national mapping agency                 |
++--------------+---------------------------------------------------------------+
+
+Landcover data
+...............
+.. image:: /static/training/socialisation/icon_landcover.png*
```
Comment (adissadis) [inline docs/source/training/socialisation/inasafe_concepts.rst]: > image:: /static/training/socialisation/icon_landcover.png*

this caused an error. It need to change into:

`image:: /static/training/socialisation/icon_landcover.*
`

**Comment by Charlotte-Morgan** [comment]
this is a part fix for #2895

File: docs/source/training/socialisation/inasafe_concepts.rst (hunk 1 above)
Comment (Charlotte-Morgan) [inline docs/source/training/socialisation/inasafe_concepts.rst]: thanks I fixed that

File: docs/source/training/socialisation/run_basic_inasafe.rst [hunk 2]
```diff
@@ -99,7 +99,7 @@ the source of the data.
 |                              |               | the Australian Government.                                                      |
 |                              |               | `See more at... <http://inasafe.org/en/training/socialisation/datasets.html>`__ |
 +------------------------------+---------------+---------------------------------------------------------------------------------+
-| Java_population_WGS84        | WorldPop      | High resolution, modelled data for human population distributions.              |
+| Jakarta_population_WGS84        | WorldPop      | High resolution, modelled data for human population distributions.              |
```
Comment (adissadis) [inline docs/source/training/socialisation/run_basic_inasafe.rst]: Found these error:

image


just need little fix backspace like this:

image

Comment (timlinux) [inline docs/source/training/socialisation/run_basic_inasafe.rst]: Lovely comments @adissadis - so nice to see such clear indications of the issue.

**Pull request merged**